src/webapp/components/DeliveryTable.js

```diff
@@ -66,6 +51,46 @@ const DeliveryTable = ({ rows }) => {
     };
   });
 
+  const renderTableHeadRow = () => {
+    return (
+      <TableRow>
+        <TableCell></TableCell>
+        <TableCell>
+          {str("webapp:deliveryNeeded.table.headers.daysOpen", {
+            defaultValue: "Days open",
+          })}
+        </TableCell>
+        <TableCell>
+          {str("webapp:deliveryNeeded.table.headers.timeline", {
+            defaultValue: "Requested Delivery Time",
+          })}
+        </TableCell>
+        <TableCell>
+          {str("webapp:deliveryNeeded.table.headers.neighbor", {
+            defaultValue: "Neighbor",
+          })}
+        </TableCell>
+        <TableCell>
+          {str("webapp:deliveryNeeded.table.headers.need", {
+            defaultValue: "Need",
+          })}
+        </TableCell>
+        <TableCell>{str("webapp:zoneFinder.label.code")}</TableCell>
+        <TableCell>
+          {str("webapp:deliveryNeeded.table.headers.crossStreets", {
+            defaultValue: "Cross streets",
+          })}
+        </TableCell>
+      </TableRow>
+    );
+  }
+
+  const renderTableBody = () => {
```

Comment:
this can be a stateless component, e.g. `const TableRows ({formattedRows}) => {...}` and you can render it below as `<TableRows />`, which is a more recent React pattern and has performance optimizations.

Reply:
you can also define it outside of `DeliveryTable` , it will be slightly more performant for re-renders since it doesn't redefine the function.